### ISSUE TITLE
Show more detail re: interest accrued (negative and positive) in the top right 

### ISSUE THREAD

**Issue by KevinRafaelConnor** [jet-lab/jet-v1#145]
From #ux-ui-improvements in Discord:

"Your deposited value" in top right should include interest accrued (positive for lending, negative for borrowing) shown as separate values

**Comment by KevinRafaelConnor** [comment]
I guess the tricky part here design-wise would be to fit in multiple assets accrued/owed interest in crypto rather than $ -- if we allow that to toggle with the crypto/USD toggle. 

Do we want to just leave this part of the screen in $ at all times to simplify? @eyevz any opinion?

**Comment by cherry-ua** [comment]
I deposited 100 USDC
Try to borrow 1 SOL 
There are no any push notification on small collateral

Please add push notification
![image](https://user-images.githubusercontent.com/81331739/140900944-54d3aef2-e7c3-4d79-95fd-c5e8005f2824.png)

**Comment by cherry-ua** [comment]
Add email notification. 
Send notification to client when C-ratio is close to 130.
It will be a respect to borrowers

**Comment by eyevz** [comment]
Once the data project is sufficiently advanced, it will be possible to add interest information. Not with respect to design, what is the information you would like to present?

These three are probably most interesting:

- Total interest earned
- Total interest paid
- Interest owed on outstanding loan

Any others?

The source data would be specific to an asset. To accumulate into USD values would require assumptions about when we measure prices.

**Comment by KevinRafaelConnor** [comment]
I think that those that you mentioned are sufficient for this ticket, and I think that having the data be asset-specific is more than fair to avoid getting into a super complex setup too soon simply to display USD. If they earn/accrue USDC it will effectively be in USD anyways, which is a big chunk of borrowing and lending. 

Just a note, this is JET-119 in Jira @eyevz 

I also created a ticket on the DATA board (DATA-10) for you, hopefully you don't mind. I left off the epic and sprint for you to specify, but figured I could help by just creating the ticket so we don't lose track.

I also linked the two issues within Jira, which is a useful function I thought you'd want to see.  DATA-10 is a blocker for JET-119 / DATA-10 blocks JET-119, and this relationship is shown in the information for both of these tickets.
